quarkchain/cluster/master.py

```diff
@@ -1286,6 +1292,12 @@ def handle_new_root_block_header(self, header, peer):
         future_list = self.broadcast_rpc(
             op=ClusterOp.ADD_ROOT_BLOCK_REQUEST, req=AddRootBlockRequest(r_block, False)
         )
+
+        for i, grpc_client in enumerate(self.grpc_slave_pool):
+            grpc_client.set_rootchain_confirmed_block(
+                self.cluster_config.GRPC_SLAVE_LIST[i].ID
```

Comment:
adding a TODO saying this is a fake message

Reply:
a second thought. we probably shouldn't use the fake message here, because it needs to be removed very soon. 

suggest we remove the `message` field in the proto.